### PR TITLE
Add: Release Watcher, Blog Watcher Fix: Documentation Searcher

### PR DESCRIPTION
**Pull request by Shawn-E**

### CODE REVIEW

**Comment by Shawn-E** [comment]
This now adds a Release Watcher and A Blog Watcher

File: Documentation Searcher/Docs Searcher.js [hunk 1]
```diff
@@ -45,7 +49,7 @@ module.exports = async function(app, connection, bot, faxstore) {
                     
                     const embed = new Discord.MessageEmbed()
                     .setColor('#1b4a6f')
-                    .setAuthor(`Documentation seach results:`)
+                    .setAuthor({ name: `Documentation seach results:` })
```
Comment (FAXES) [inline Documentation Searcher/Docs Searcher.js:52]: This is the only line that really needs changing — no point in running a forEach function for one command. The `bootFaxStoreBot` is also recommended so the command is always registered.

**Review by FAXES — CHANGES_REQUESTED** [review]
See attached comments.

File: Documentation Searcher/Docs Searcher.js (hunk 1 above)
Comment (Shawn-E) [reply Documentation Searcher/Docs Searcher.js:52]: The reason I did the for each was due to the commands not registering into the bot and what seems to be the issue is the `bootFaxStoreBot` function pictures below

https://cdn.shawnengmann.com/shawn/_DaE-v.png
https://cdn.shawnengmann.com/shawn/TU3PBw.png

**Pull request closed without merging**